dependencies/pom.xml

```diff
@@ -501,6 +502,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>net.bytebuddy</groupId>
+                <artifactId>byte-buddy</artifactId>
+                <version>${version.lib.bytebuddy}</version>
+            </dependency>
```

Comment:
@barchetta to review this

Reply:
Do we use this dependency directly (I do not see where we do) or are we managing the version of a transitive dependency? If the later, is this for dependency convergence? Forced upgrade? I don't see an issue here, I just want to understand why. Thx.

Reply:
Dependency convergence - mockito and hibernate.